src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    setStatus: (state) => state.status,
+  },
```

Comment:
- [ ] It will be great to ensure that your reducer checks the status. At the moment, the only thing that is doing is just returning `under construction` without any form of check. For example ⬇️ 
```
 reducers: {
    checkStatus: (state, action) => {
      state.status =
        action.payload === "Under construction"
          ? "Under construction"
          : state.status
    },
```